.travis.yml

```diff
@@ -17,21 +24,160 @@ gemfile:
   - gemfiles/4.0.gemfile
   - gemfiles/4.1.gemfile
   - gemfiles/4.2.gemfile
+  - gemfiles/5.0.gemfile
+  - gemfiles/5.1.gemfile
+  - gemfiles/5.2.gemfile
+  - gemfiles/6.0.gemfile
+  - gemfiles/6.1.gemfile
+  - gemfiles/7.0.gemfile
+  - gemfiles/7.1.gemfile
```

Comment:
i added 7.0 and 7.1 after fixing 6.1. and to my surprise it all worked without additional changes